.tekton/integration/integration-test.yaml

```diff
@@ -0,0 +1,192 @@
+kind: Pipeline
+apiVersion: tekton.dev/v1beta1
+metadata:
+  name: integration-tests
```

Comment:
Do we control what version of OCP this runs on?

Reply:
4.19